bower.json

```diff
@@ -28,6 +28,7 @@
     "js-message": "latest",
     "event-pubsub": "latest",
     "browser-error-classes": "latest",
-    "document-register-element": "latest"
+    "document-register-element": "latest",
+    "highcharts": "latest"
```

Comment:
![dont use latest](https://cdn.meme.am/instances/500x/66523697.jpg)

for highcharts only